pkg/build/controller/build/build_controller.go

```diff
@@ -1268,25 +1308,6 @@ func (bc *BuildController) createBuildPod(build *buildv1.Build) (*buildUpdate, e
 				return update, err
 			}
 		}
-
-	} else {
```

Comment:
@ricardomaraschini we still need this block to handle the case where the build pod was created, but the child ConfigMaps have not all been created yet.

Reply:
This is still there but in a different position. This branch now is executed when we know that the pod does not exist(`errors.IsNotFound(err)`) on https://github.com/openshift/openshift-controller-manager/pull/39/files#diff-4349d2367514c13a1d5c76dc4debf58fR1218